codec-native-quic/src/main/c/netty_quic_boringssl.c

```diff
@@ -185,9 +185,15 @@ static STACK_OF(CRYPTO_BUFFER)* arrayToStack(JNIEnv* env, jobjectArray array, CR
             CRYPTO_BUFFER_free(buffer);
             goto cleanup;
         }
+        (*env)->ReleaseByteArrayElements(env, bytes, (jbyte*)data, 0);
+        (*env)->DeleteLocalRef(env, bytes);
     }
     return stack;
 cleanup:
+    for (int j = 0; j < sk_CRYPTO_BUFFER_num(stack); j++) {
+        CRYPTO_BUFFER *buffer = sk_CRYPTO_BUFFER_value(stack, j);
+        CRYPTO_BUFFER_free(buffer);
+    }
```

Comment:
```suggestion
```
Revert this... `sk_CRYPTO_BUFFER_pop_free(stack, CRYPTO_BUFFER_free);` will free all elements.

Reply:
@wuhaocn please address this ...